client/src/app/vets/components/veterinarians/veterinarians.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="main_veterenarians">
+  <h2>Наші ветеринари</h2>
+</div>
+<section class="veterenarians-section">
```

Comment:
The same question as previously...

Reply:
It wasn`t my task, it is an independent task that other team member was assigned